sass/base/_base.scss

```diff
@@ -0,0 +1,31 @@
+*,
+*::after,
+*::before {
+	margin: 0;
+	padding:0;
+	box-sizing: inherit;
+	}
+
+html {
+	font-size: 62.5%;
+	background-color: $color-light;
+	scroll-behavior: smooth;
+	@include respond(tab-land) { //witdh < 1200px
+		font-size: 56.25%; // 1 rem = 9px, 9/16 = 56.25%
+	}
+
+	@include respond(tab-port) { //width < 900px
```

Comment:
Te komentarze mogą kłamać :) Lepiej na nich nie polegać i wywalić. Jaki tutaj miałeś problem, że wymagało to komentarza? ` //width < 900px`